src/assets/stylesheets/shared/_layout.scss

```diff
@@ -237,8 +237,7 @@ footer
 }
 
 .btn{
-  min-width: 90px;
-  font-size: $font-size-base;
```

Comment:
i believe the font size is, but is not being applied to the buttons. please verify that all of the buttons still have 14pixel font with this change.